apps/web/components/Proposals.tsx

```diff
@@ -308,15 +318,16 @@ export function Proposals({
 
   // console.log("newLocal:                    %s", memberActivatePointsAsNum);
   // console.log("newLocal_1:                  %s", totalEAPasNum);
-  console.log("memberActivatedPoints:       %s", memberActivatedPoints);
-  console.log("memberPoolWeight:            %s", memberPoolWeight);
+  // console.log("memberActivatedPoints:       %s", memberActivatedPoints);
+  // console.log("memberPoolWeight:            %s", memberPoolWeight);
   // console.log(
   //   "totalEffectiveActivePoints:  %s",
   //   strategy.totalEffectiveActivePoints,
   // );
   // console.log("tokenDecimals:               %s", tokenDecimals);
 
   const calcPoolWeightUsed = (number: number) => {
+    if (memberPoolWeight == 0) return 0;
```

Comment:
always use brackers in if `{´ better always be explicit about it

Reply:
i love this more compact way of writing code, but i will change it if you think is better for the whole team to understand.

Reply:
There 2 problems, first `return` before anything can lead a problem in refactor, also for fast scanner with brackets its better.

For check diffs too